apps/explorer-e2e/src/support/pages/home-page.js

```diff
@@ -1,3 +1,67 @@
 import BasePage from './base-page';
+export default class HomePage extends BasePage {
+  statsEnvironmentTitle = 'stats-environment';
+  statsTitle = 'stats-title';
+  statsValue = 'stats-value';
 
-export default class HomePage extends BasePage {}
+  verifyStatsEnvironment(expectedEnv) {
+    cy.getByTestId(this.statsEnvironmentTitle).should('have.text', expectedEnv);
+  }
+
+  verifyStatsTitlesDisplayed() {
+    const statTitles = [
+      'Status',
+      'Height',
+      'Validating nodes',
+      'Uptime',
+      'Total nodes',
+      'Inactive nodes',
+      'Total staked',
+      'Backlog',
+      'Trades / second',
+      'Orders / block',
+      'Orders / second',
+      'Transactions / block',
+      'Block time',
+      'Time',
+      'App',
+      'Tendermint',
+      'Up since',
+      'Chain ID',
+    ];
+
+    cy.getByTestId(this.statsTitle)
+      .each(($title, index) => {
+        cy.wrap($title).should('have.text', statTitles[index]);
+      })
+      .then(($list) => {
+        cy.wrap($list).should('have.length', 18);
+      });
+  }
+
+  verifyStatsValuesdisplayed() {
+    cy.getByTestId(this.statsValue)
+      .each(($value) => {
+        cy.wrap($value).should('not.be.empty');
+      })
+      .then(($list) => {
+        cy.wrap($list).should('have.length', 18);
+      });
+  }
+
+  verifyStatsBlockHeightUpdating() {
+    cy.getByTestId(this.statsValue)
+      .eq(1)
+      .invoke('text')
+      .then((blockHeightTxt) => {
+        cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
+        //Wait needed to allow blocks to change
```

Comment:
chore (blocking): can we not poll for this in some way rather than waiting a specific amount of time and let the test timeouts catch it 🤔

Reply:
Possibly intercepts are the way to go here, if you intercept the `statistics` query you can alias that and wait for that 👍

Reply:
I messed around with intercept and found out Cypress was already waiting properly without the implicit wait 🤦‍♂️. It wasn't all wasted effort though. I've replaced a wait elsewhere with an intercept 👍🏼

Reply:
Nice!